### DISCUSSION TITLE
reaction fingerprint agent information

### DISCUSSION THREAD

**Discussion by manajit-das** [rdkit/rdkit#5791] [Q&A] [answered]
In the current implementation of CreateDifferenceFingerprintForReaction function, Can I  include the agents information as shown in the paper https://pubs.acs.org/doi/10.1021/ci5006614
If not how to include it?
Thank you so much.

**Comment by liudongliangHI** [comment, marked as answer]
yes，you can use ReactionFingerprintParams class for the agents information. see the RDKit documentation
[ReactionFingerprintParams](http://rdkit.org/docs/source/rdkit.Chem.rdChemReactions.html?highlight=reactionfingerprintparams#rdkit.Chem.rdChemReactions.ReactionFingerprintParams)

This use case notebook maybe useful for you:
[Create-reaction-difference-FPs](https://github.com/hcji/ChemReactionClassification/blob/9a3eb0e26883180719db8928e22db78310a28686/notebooks/Create-reaction-difference-FPs.ipynb)

**Reply by manajit-das** [reply]
Thank you @liudongliangHI , It works.
